src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -9,14 +9,6 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Constants
 
-(def required-keys
-  "Keys required in an auth rule map."
-  [:path :type])
-
-(def required-or-key
-  "At least one of these keys is required in an auth rule map."
-  #{:deny :allow})
-
```

Comment:
I inlined these in the `validate-auth-config-rule!` function because the exception message(s) contains hardcodings for these keys anyway, so I don't see any value in pulling these out.